editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2640,6 +2640,36 @@ void CanvasItemEditor::_update_cursor() {
 	set_default_cursor_shape(c);
 }
 
+void CanvasItemEditor::_update_lock_and_group_button() {
+	bool all_locked = true;
+	bool all_group = true;
+	List<Node *> selection = editor_selection->get_selected_node_list();
+	if (selection.is_empty()) {
+		all_locked = false;
+		all_group = false;
+	} else {
+		for (Node *E : selection) {
```

Comment:
why not use one for loop?

Reply:
We can use one loop of course, just copied and pasted from the origin code.